.github/workflows/release-please.yml

```diff
@@ -0,0 +1,27 @@
+name: release-please
+
+on:
+  push:
+    branches:
+      - staging
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: GoogleCloudPlatform/release-please-action@v3.7.13
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          release-type: simple
+          changelog-types: |
+            [
+              {"type":"feat","section":"Nouveautés","hidden":false},
+              {"type":"refactor","section":"Améliorations","hidden":false},
```

Comment:
Je le trouve pas intuitif que refactor = Améliorations. Dans ma tête refactor c'est un changment du code sans chngement pour les utilisateurs et améliorations c'est des améliorations du produit qui impactent l'utilisateur

Reply:
en effet y'a 2 possibilités : 
- soit on rebascule `refactor` dans Technique, et on regroupe les Nouveautés et Améliorations via `feat`
- soit on utilise d'autres tokens comme `perf` pour du refactoring pure code

Reply:
Dans tous les cas, je pense qu'on aura toujours une petite étape pour retravailler la release une fois publiée dans github, pour ensuite la copier-coller et la partager au reste de l'équipe. Et à ce moment là séparer les Nouveautés des Améliorations, ou au moins regrouper par fonctionnalité

Reply:
il y a une discussion ici pour rajouter `improve` par exemple : https://github.com/conventional-commits/conventionalcommits.org/issues/66
je vais voir si on peut le rajouter as-is

Reply:
en tout cas la Github Action https://github.com/amannn/action-semantic-pull-request a l'air de pouvoir customiser la liste des des "types"

Reply:
du coup je vais : 
- basculer "refactor" dans la section Technique
- merger la PR
- voir ce que ca donne, en discuter avec l'équipe, et refaire une PR supplémentaire pour rajouter "improve"

Reply:
(dernière) PR ouverte ici : https://github.com/betagouv/ma-cantine/pull/4758